### ISSUE TITLE
running antiSMASH error

### ISSUE THREAD

**Issue by Rahimlou** [nextgenusfs/funannotate#819]
I get the following error by running antiSMASH using "funannotate remote" command. 

from Bio.Align import _aligners
ImportError: cannot import name '_aligners' from 'Bio.Align'

my code: 
module load py-antismash/6-1-1
funannotate remote -i Gene_Prediction -m antismash -e email@server.edu -o xxx

**Comment by hyphaltip** [comment]
can you check what version of biopython you have installed?

**Comment by nextgenusfs** [comment]
Not sure what your system setup is, but `funannotate remote` would send the data to the antiSMASH service -- I think this might be broken with update to version 6.  But if you are loading a module here and trying to run locally -- that is not what `funannotate remote` does -- it was originally setup as a convenience when installing antismash locally was not possible. Recommended way would now be to run antismash locally via ronda-mediated install (separate environment) and then you an pass those data to `funannotate annotate` via the `--antismash` cmd line option.

**Comment by Rahimlou** [comment]
> Not sure what your system setup is, but `funannotate remote` would send the data to the antiSMASH service -- I think this might be broken with update to version 6. But if you are loading a module here and trying to run locally -- that is not what `funannotate remote` does -- it was originally setup as a convenience when installing antismash locally was not possible. Recommended way would now be to run antismash locally via ronda-mediated install (separate environment) and then you an pass those data to `funannotate annotate` via the `--antismash` cmd line option.

Exactly, this is the best idea to run antismash individually. I could install the antismash and downloaded the relevant databases. May I know which files from the gene prediction step I should use as the input to antismash?
I used .gbk and .gff3, but I got the following error:
ValueError: Need a Nucleotide or Protein alphabet

**Comment by nextgenusfs** [comment]
You would pass the GBK file from predict or update (if you had RNA seq).

**Comment by Rahimlou** [comment]
> 

Yes, I have RNA seq. How can I update it? Is there a manual to do it?
